### PR TITLE
browsers: add arc

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

File: browserexport/browsers/all.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-from .chromium import Chromium
-from .brave import Brave
-from .safari import Safari
-from .vivaldi import Vivaldi
-from .firefox_mobile import FirefoxMobile
-from .firefox_mobile_legacy import FirefoxMobileLegacy
+from browserexport.browsers.common import Browser
+from browserexport.browsers.firefox import Firefox
+from browserexport.browsers.waterfox import Waterfox
+from browserexport.browsers.palemoon import Palemoon
```
Comment (AndrewSB) [inline browserexport/browsers/all.py:5]: whoops. this seems to have happened automatically from the formatter, want me to revert?

Comment (seanbreckenridge) [reply browserexport/browsers/all.py:5]: Yeah, should be relative imports inside the package

Comment (AndrewSB) [reply browserexport/browsers/all.py]: done, i think my editor defaults to using black to format, which seems incompatible with setup.cfg. 

for new projects, are you using the pyproject.toml thing they recommend?

Comment (seanbreckenridge) [reply browserexport/browsers/all.py]: I use black as well, should work [fine with setup.cfg](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html) as long as its running the formatter from the root dir -- I don't have anything special in the setup.cfg for black though, is all just the defaults

Comment (seanbreckenridge) [reply browserexport/browsers/all.py]: Ah nevermind, I misread, it does only use pyproject.toml

But not using it anyways, so must be something else changing the imports to absolute, unless you have a global black configuration file

**Comment by seanbreckenridge** [comment]
Assuming this has no database schema changes from chromium, looks good, thanks

**Comment by seanbreckenridge** [comment]
Pushed new version (0.2.8) to pypi

File: browserexport/browsers/all.py (hunk 1 above)
Comment (AndrewSB) [reply browserexport/browsers/all.py]: weird yeah no idea, i don't have a config, the only python i write is in colab notebooks usually.

thanks, i'll test this and propose changes if it doesn't work

**Pull request merged**